.github/workflows/go.yml

```diff
@@ -64,6 +64,10 @@ jobs:
               working-directory: ./go
               run: make install-tools-go${{ matrix.go }}
 
+            - name: Set LD_LIBRARY_PATH
+              if: ${{ matrix.os == 'ubuntu-latest' }}
+              run: echo 'LD_LIBRARY_PATH="${{ GITHUB_WORKSPACE }}/go/target/release/deps/' >> $GITHUB_ENV
```

Comment:
Probably you need to extend it instead of overwritting: `PATH=$PATH:/new/path`

Reply:
Yeah I was wondering about that, currently the path is not set on ubuntu GH runners but it would be better to extend it. Wanted to see if I could get the CI to be green first, but now that it is I'll do this